pkg/kubelet/kubelet_volumes.go

```diff
@@ -128,6 +128,18 @@ func (kl *Kubelet) cleanupOrphanedPodDirs(pods []*v1.Pod, runningPods []*kubecon
 			orphanVolumeErrors = append(orphanVolumeErrors, fmt.Errorf("Orphaned pod %q found, but volume paths are still present on disk", uid))
 			continue
 		}
+
```

Comment:
In the above podVolumesExist() checking, it calls getMountedVolumePathListFromDisk() to check whether there are mounted volumes exist. And then it calls getPodVolumePathListFromDisk to check whether there are still volume paths exist. I think there are some redundancy here.

Reply:
it might be redundant, but I don't want to remove it as part of this bug fix, as this code is not very well tested.

Reply:
I'll leave it comment that this may be able to be cleaned up

Reply:
Opened up https://github.com/kubernetes/kubernetes/issues/72346